source/hands_on/her.rst

```diff
@@ -53,17 +53,14 @@ Benchmark
      - best mean reward
      - evaluation results
      - config link
-   ..   - comparison
    * - BitFLip-15bit
      - 0.9
      - .. image:: images/benchmark/bitflip15bit-herdqn.png
      - `config_link <https://github.com/opendilab/DI-engine/blob/main/dizoo/classic_control/bitflip/config/bitflip_her_dqn_config.py>`_
-   ..   - | Tianshou(20)
    * - BitFLip-20bit
      - 0.9
      - .. image:: images/benchmark/bitflip20bit-herdqn.png
      - `config_link <https://github.com/opendilab/DI-engine/blob/main/dizoo/classic_control/bitflip/config/bitflip_her_dqn_config.py>`_
-   ..   - | Tianshou(16245)
 
 
 P.S.
```

Comment:
i can‘t see any benchmark pics when i view this file，Is this because of previrew on github？

Reply:
Yes, you can render it by running ``make html`` at root path of ``DI-engine-docs``